src/RecoveryServices/RecoveryServices.Backup/Cmdlets/ProtectionPolicy/GetAzureRmRecoveryServicesBackupProtectionPolicy.cs

```diff
@@ -58,7 +58,7 @@ public class GetAzureRmRecoveryServicesBackupProtectionPolicy : RSBackupVaultCmd
         /// Backup management type of the policy to be fetched
         /// </summary>
         [Parameter(ParameterSetName = WorkloadBackupMangementTypeParamSet, Position = 3,
-            Mandatory = true, HelpMessage = ParamHelpMsgs.Common.BackupManagementType)]
+            Mandatory = false, HelpMessage = ParamHelpMsgs.Common.BackupManagementType)]
```

Comment:
WorkloadBackupMangementTypeParamSet and WorkloadParamSet are identical after this change, please remove WorkloadBackupMangementTypeParamSet  if it's not needed

Reply:
reverted this changed due to same reason - we need both param sets.